Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM debian:jessie
+FROM gliderlabs/alpine:3.3
 
-COPY bin/resolve-ip /usr/bin/resolve-ip
-COPY GeoLiteCity /usr/bin/GeoLiteCity
+COPY GeoLiteCity /bin/GeoLiteCity
```

Comment:
I think you need to put this in `/GeoLiteCity` since the workdir by default is `/`